ui/components/app/modals/confirm-delete-rpc-url-modal/confirm-delete-rpc-url-modal.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  BlockSize,
+  Display,
+} from '../../../../helpers/constants/design-system';
+import {
+  Box,
+  ButtonPrimary,
+  ButtonPrimarySize,
+  ButtonSecondary,
+  ButtonSecondarySize,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+} from '../../../component-library';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import {
+  hideModal,
+  setEditedNetwork,
+  toggleNetworkMenu,
+} from '../../../../store/actions';
+
+const ConfirmDeleteRpcUrlModal = () => {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+  return (
+    <Modal
+      isClosedOnEscapeKey={true}
+      isClosedOnOutsideClick={true}
+      isOpen={true}
```

Comment:
I don't think `={true}` is required here; I think the presence of the attribute infers `true`